ovos_utils/gui.py

```diff
@@ -79,6 +79,29 @@ def can_use_gui(bus=None,
     return can_use_local_gui() or is_gui_connected(bus)
 
 
+def get_ui_directories(root_dir: str) -> dict:
+    """
+    Get a dict of available UI directories by GUI framework.
+    @param root_dir: base directory to inspect for available UI directories
+    @return: Dict of framework name to UI resource directory
+    """
+    ui_directories = dict()
+    base_directory = root_dir
+    if isdir(join(base_directory, "gui")):
+        LOG.debug("Skill implements resources in `gui` directory")
+        ui_directories["all"] = join(base_directory, "gui")
+        return ui_directories
+    LOG.info("Checking for legacy UI directories")
+    if isdir(join(base_directory, "ui5")):
```

Comment:
this was never formally introduced, we can just drop it, ui5 and ui6 dont exist in the wild

Reply:
Okay, saw it in some places but I wasn't sure if it actually got implemented or just in PRs. Better to never support it then keep it around as deprecated